### ISSUE TITLE
add a gitcoin.co/explorer stale bounty filter

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#7195]
##### Description 

Gitcoin.co/explorer lists out the all the bounties available at the moment 
Many times there are bounties which get old and the funder loses track of them 

The requirement is to add a filter similar to [THIS](https://user-images.githubusercontent.com/5358146/90024715-d00d5900-dcd2-11ea-96ec-92c5f8bd195f.png) in the [My bounties section](https://user-images.githubusercontent.com/5358146/90024429-79078400-dcd2-11ea-9983-3049098b7c4f.png)

In other words,
If a funder is logged in ->
they should be able to view bounties created by them more than 3 days but which are still in open state
